static/js/notifications.js

```diff
@@ -277,6 +277,38 @@ exports.notify_above_composebox = function (note, link_class, link_msg_id, link_
     $('#out-of-view-notification').show();
 };
 
+// Set the inline_notification_flow_supported flag to notify
+// desktop app that the current version of zulip app will send inline
+// notification data; the code that sends the data is in process_notification
+// function below. And also listen for send_inline_notification_message.
+if (window.electron_bridge !== undefined) {
+    window.electron_bridge.inline_notification_flow_supported = true;
+
+    // This is done so desktop app doesn't have to depend on channel.post
+    // for setting crsf_token and narrow.by_topic to narrow to the message
+    // being sent.
+    window.electron_bridge.on_event('send_inline_notification_message', function (message_id, data) {
+        function success() {
+            narrow.by_topic(message_id);
```

Comment:
We should probably pass `{trigger: 'desktop_inline_reply'}` as the second argument here; it's mostly instrumentation.

Reply:
Done!